typescript/nomad-sdk/src/utils.ts

```diff
@@ -99,3 +106,28 @@ export function evmId(data: BytesLike): Address {
 export function delay(ms: number): Promise<void> {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
+
+/**
+ * Wrap an ethers Signer in a Gnosis Safe signer
+ * @param safeAddress The address of the Gnosis Safe
+ * @param signer The ethers signer to wrap. This signer SHOULD have ownership
```

Comment:
note: this will need to be a delegate, rather than a signer on the safe. we don't want to export the keys of actual signers on the safe

Reply:
should be fixed in the next PR, when we get the end-to-end flow working

Reply:
agree on next pr